geometric_kernels/kernels/geometric_kernels.py

```diff
@@ -62,13 +66,7 @@ def init_params_and_state(self):
         :return: tuple(params, state)
         """
         params = dict(lengthscale=np.array(1.0), nu=np.array(np.inf))
-
-        eigenvalues_laplacian = self.space.get_eigenvalues(self.num_eigenfunctions)
-        eigenfunctions = self.space.get_eigenfunctions(self.num_eigenfunctions)
-
-        state = dict(
-            eigenvalues_laplacian=eigenvalues_laplacian, eigenfunctions=eigenfunctions
-        )
+        state = dict()
```

Comment:
Maybe we should actually use this opportunity to break backwards-compatibility early on (can discuss on the next meeting)

Reply:
Could you then take over and make sure all the notebook examples are updated accordingly?

Reply:
Yes, I am revising the example notebooks anyway